### PR TITLE
Routes update for new page only

### PR DESCRIPTION
**Pull request by izzyL-DJ**

### CODE REVIEW

File: server/routes/info.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { Router } from 'express'
+
+export default function infoRoutes(router: Router) {
+  router.get('pages/info', (req, res) => {
```
Comment (marcus-bcl) [inline server/routes/info.ts:4]: This one is the URL path, should it be something like `/tiering-information` ?  I'd have suggested `/info` but that's taken - https://tier-dev.hmpps.service.justice.gov.uk/info

Also I guess you're planning to add the `pages/info.njk` file later?

Comment (izzyL-DJ) [reply server/routes/info.ts:4]: Ah okay didn't know that page already existed.

Yeah I'm planning to add the info.njk file later, just wanted to sort out the routes first

Comment (izzyL-DJ) [reply server/routes/info.ts]: Changed the reference to `/tiering-additional-information`. Also I did have it without the `pages/` path but still ran into a 404

File: server/routes/tiering-additional-information.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { Router } from 'express'
+
+export default function tierAddInfoRoutes(router: Router) {
+  router.get('/tiering-additional-information', (req, res) => {
+    res.render('/tiering-additional-information')
```
Comment (marcus-bcl) [inline server/routes/tiering-additional-information.ts:5]: You'll still need this one to be `/pages/info` - the first string is the URL path, the second is the path to your .njk file.

Comment (izzyL-DJ) [reply server/routes/tiering-additional-information.ts]: Took that out by accident 😬

**Review by marcus-bcl — APPROVED** [review]

**Comment by izzyL-DJ** [comment]
Removed, reloaded, then added the `pages/` part for the render and it magically works now 🤔

File: server/views/pages/tiering-additional-information.njk [hunk 3]
```diff
@@ -0,0 +1,555 @@
...
+                </div>
+            </div>
+        <br/>
+        <br/>
+
+        <h2 class="govuk-heading-m" id="weight-methodology">Development of weightings methodology</h2>
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-full">
+                    <p>The Unified Tiering Model (UTM) requires weightings to feed other organisational tools such as the Workload Measurement Tool (WMT).
+                        (Current Kainos WMT will be replaced by MoJ WMT by Dec 2021)
```
Comment (marcus-bcl) [inline server/views/pages/tiering-additional-information.njk:445]: Pretty sure this never happened 😄  we should remove this line.

File: server/views/pages/tiering-additional-information.njk [hunk 4]
```diff
@@ -0,0 +1,555 @@
...
+                        <li>Phase 2: New weightings methodology, based on key activity timings, to be embedded through an iterative process
+                        </li>
+                    </ul>
+                    <img src="../../../assets/images/dev_of_weightings.png" alt="A diagram detailing the development plan for phase 1 and 2" height="450" width="1000">
+                </div>
+            </div>
+        <br/>
+        <br/>
+
+        <h2 class="govuk-heading-m" id="unified-weightings">Unifited Weightings - Principles</h2>
```
Comment (marcus-bcl) [inline server/views/pages/tiering-additional-information.njk:463]: Typo
```suggestion
        <h2 class="govuk-heading-m" id="unified-weightings">Unified Weightings - Principles</h2>
```

File: server/views/pages/tiering-additional-information.njk [hunk 5]
```diff
@@ -0,0 +1,555 @@
...
+                <div class="govuk-grid-column-full">
+                    <p>The Unified Tiering Model (UTM) requires weightings to feed other organisational tools such as the Workload Measurement Tool (WMT).
+                        (Current Kainos WMT will be replaced by MoJ WMT by Dec 2021)
+                    </p><br/>
+                    <p>Plan is to move beyond dated probation modelling (used in the NPS Interim Tiering Model) and take a new approach using a
+                        renewed activity-based methodology (utilising learning from the OMiC model).
+                    </p><br/>
+                    <strong>2-phase approach to be taken:</strong>
+                    <ul class="govuk-list--bullet">
+                        <li>Phase 1: To facilitate launch of UTM in April 21, will utilise current ITM weightings methodology with additional logic
```
Comment (marcus-bcl) [inline server/views/pages/tiering-additional-information.njk:452]: Maybe remove this whole section actually - as the work has now been done.

**Review by marcus-bcl — APPROVED** [review]
Looks good - approved with a couple of minor comments.

File: server/views/pages/tiering-additional-information.njk (hunk 3 above)
Comment (izzyL-DJ) [reply server/views/pages/tiering-additional-information.njk:445]: Ah okay. This was lifted directly from the powerpoint

File: server/views/pages/tiering-additional-information.njk (hunk 5 above)
Comment (izzyL-DJ) [reply server/views/pages/tiering-additional-information.njk:452]: Would that be from 441? Or just the part about the 2-phase approach?

**Review by marcus-bcl — APPROVED** [review]

**Pull request merged**